src/Command/Process/Action/ReviewAppliedAction.php

```diff
@@ -73,7 +73,7 @@ function ($patch) {
             $appliedOptionalPatches
         );
         $totalNumber = count(array_unique(array_merge($ids, $patchFilter)));
-        if ($totalNumber >= self::UPGRADE_THRESHOLD) {
+        if ($totalNumber >= self::UPGRADE_THRESHOLD && $input->getOption('format') !== 'json') {
```

Comment:
```suggestion
        if ($totalNumber >= self::UPGRADE_THRESHOLD) {
```
This check can be avoided here, if you won't call `   $this->reviewAppliedAction->execute($input, $output, []);` when format == 'json' in [Command/Process/ShowStatus.php](https://github.com/magento/magento-cloud-patches/pull/86/files#diff-9d93ba3f4827d2759e1ef5a0b570cd70d9b30399619fe53d72260d5dd87101c1)